components/Alert/Alert.tsx

```diff
@@ -21,16 +22,20 @@ const alertIconMap: { [type: string]: string } = {
 }
 
 const Alert: React.FC<Props> = ({ alert, onDismiss }) => {
+  const [show, setShow] = useState(true)
+
   const { text, type, url, urlCaption, id } = alert
   const icon = alertIconMap[_.defaultTo(type, '-1')]
-  const alertClasses =
-    type === 'urgent' ? 'alert alert-danger' : `alert alert-primary`
   const imageClass = type === 'urgent' ? '' : styles.info_image
 
-  return (
-    <div
-      className={`${styles.alert} d-flex justify-content-between mt-3 ${alertClasses}`}
-      role="alert"
+  return show ? (
+    <AlertBS
+      variant={`${type === 'urgent' ? 'danger' : 'primary'}`}
+      onClose={() => {
+        setShow(false)
+        onDismiss && onDismiss(id)
```

Comment:
Just out of curiosity and to confirm, does `onDismiss` handle whether or not the alert will be displayed for particular user in the future(if the user clicks the x button to dismiss)?

Reply:
No, it doesn't handle it. It gets called when the alert is closed. It's useful when we want to perform a certain action when the alert closes.